### DISCUSSION TITLE
Ability to provide human readable alias to influx buckets

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana#48730] [General]
Greetings,

When creating influxdb data source, provide ability to give alias to bucket names. That way you can display them in a drop down with clean human readable values that can query for creation of variables

https://community.grafana.com/t/display-name-for-dashboard-variables-vs-actual-values/65034/4

Thanks

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
